referencia/webpayrest/README.md

```diff
@@ -3053,25 +3049,15 @@ installments_number  <br> <i> Number </i> | Cantidad de cuotas. Largo máximo:
 **Respuesta**
 
 ```java
```

Comment:
Maybe this should be http?
```suggestion
```http
```

Reply:
I will ignore this, since this is the Java snippet and i didn't touched this line xD